### ISSUE TITLE
NullPointerException right when starting the sample

### ISSUE THREAD

**Issue by AndroidDeveloperLB** [danoz73/RecyclerViewFastScroller#23]
I keep getting this exception:
```java
java.lang.NullPointerException: Attempt to invoke interface method 'float xyz.danoz.recyclerviewfastscroller.calculation.progress.TouchableScrollProgressCalculator.calculateScrollProgress(android.support.v7.widget.RecyclerView)' on a null object reference
 	at xyz.danoz.recyclerviewfastscroller.AbsRecyclerViewFastScroller$1.onScrolled(AbsRecyclerViewFastScroller.java:185)
 	at android.support.v7.widget.RecyclerView.notifyOnScrolled(RecyclerView.java:3248)
 	at android.support.v7.widget.RecyclerView.dispatchLayout(RecyclerView.java:2229)
```

I've used Eclipse since for some reason I can't import the project using Android-Studio (wrote about it here: https://github.com/danoz73/RecyclerViewFastScroller/issues/19 ) .

**Comment by AndroidDeveloperLB** [comment]
Why closed? was it fixed?

**Comment by danoz73** [comment]
Nope! Sorry got this confused with the common NPE the was just fixed.

**Comment by georgelin422** [comment]
got the same problem here:

```
java.lang.NullPointerException: Attempt to invoke interface method 'float xyz.danoz.recyclerviewfastscroller.calculation.progress.TouchableScrollProgressCalculator.calculateScrollProgress (android.support.v7.widget.RecyclerView)' on a null object reference
```

**Comment by danoz73** [comment]
[Fixed here](7c3dbcde651425e1372782666f5f71e48360d392) and in v0.1.3

**Comment by AndroidDeveloperLB** [comment]
Thank you.
